esp/esp/program/modules/handlers/schedulingcheckmodule.py

```diff
@@ -553,3 +554,33 @@ def special_classroom_types(self):
          return self.formatter.format_table(mismatches,
                                             {'headings': HEADINGS},
                                             help_text=self.special_classroom_types.__doc__)
+
+     # This isn't really a scheduling check. It's a check that's useful
+     # to run before scheduling. But it works well with the format and
+     # this way everyone else doesn't have to rediscover the round_to
+     # argument to ESPUser.getTaughtTime() every year.
```

Comment:
Yeah, I kinda suspected that, it's buried pretty deep in ESPUser and I don't remember ever noticing it before.